Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -91,12 +91,12 @@ def tests
       end
 
       args = ["-I", HOMEBREW_LIBRARY_PATH/"test"]
-      args += %w[
+      args += %W[
         --color
         --require spec_helper
         --format progress
         --format ParallelTests::RSpec::RuntimeLogger
-        --out tmp/parallel_runtime_rspec.log
+        --out #{ENV["HOME"]}/.cache/parallel_rspec/runtime.log
```

Comment:
Ok, could we use `HOMEBREW_CACHE` then? I really don't want yet another dotfile in my home directory. Any we can still set `HOMEBREW_CACHE` to `$HOME/.cache` on Travis.